blocks/header/breadcrumb-details.md

```diff
@@ -0,0 +1,257 @@
+# Breadcrumbs Implementation - Updated
```

Comment:
what is all of this -- do we really need it? You can't speak to design decisions, only the customer can. And do we really need all the testing info for breadcrumbs? I would just add the bit about the "set breadcumbs=true if yuo want them" in the Library document and get rid of all this.